src/services/signature-check.ts

```diff
@@ -0,0 +1,17 @@
+import * as JWT from "jsonwebtoken";
+import {getCertificateChain} from "./azure";
+
+export const checkSignature = async (encodedToken: string, decodedToken: any): Promise<void> => {
+  // tid = tenant ID, kid = key ID
+  const certificate = await getCertificateChain(decodedToken.payload.tid, decodedToken.header.kid);
+
+  JWT.verify(
+    encodedToken,
+    certificate,
+    {
+      audience: decodedToken.payload.aud,
+      issuer: decodedToken.payload.iss,
```

Comment:
nice stuff, I think we will have to update the DevOps to let them know they can remove the old config objet in SM since it is now given as part of the token when validating the certificate and token. I believe that before the issuer was part of SM.